proj/src/main/webapp/script.js

```diff
@@ -144,24 +144,64 @@ $('#collapse-icon').addClass('fa-angle-double-left');
 
 // Collapse on click
 $('[data-toggle = sidebar-colapse]').click(function() {
-    SidebarCollapse();
+    sidebarCollapse();
 });
 
-// Currently hides the sidebar on smaller and medium screens (TODO: adjust screen for different screen sizes)
-function SidebarCollapse () {
+function sidebarCollapse () {
+    // remove locational reset 
+    $('#bottom').removeClass('topbtn'); 
+
+    // if the device is small, this will hide the chat when they open the side bar
+    if (screen.width < 750) {
+        $('.sidebar + .p-4').toggleClass('d-none'); 
+    }
+
+    // collapse sidebar as normal 
     $('.menu-collapsed').toggleClass('d-none');
     $('.sidebar-submenu').toggleClass('d-none');
     $('.submenu-icon').toggleClass('d-none');
     $('#sidebar-container').toggleClass('sidebar-expanded sidebar-collapsed');
-    
+
     // Treating d-flex/d-none on separators with title
     var SeparatorTitle = $('.sidebar-separator-title');
     if ( SeparatorTitle.hasClass('d-flex') ) {
         SeparatorTitle.removeClass('d-flex');
     } else {
         SeparatorTitle.addClass('d-flex');
     }
-    
-    // Collapse/Expand icon
-    $('#collapse-icon').toggleClass('fa-angle-double-left fa-angle-double-right');
-}
\ No newline at end of file
+
+    // move the buttom on mobile view
+    if ((screen.width<500) && ($( "#sidebar-container" ).hasClass( "sidebar-collapsed" ))) {
```

Comment:
500 seems like a bit of a magic number for a .js file. Is that the size of a phone or something? It might be good to have this in a constant somewhere so it can be referenced wherever it is needed. But not necessary.

Reply:
The Pixel phone has a width of 411px so I rounded it to 500px for the mobile view